docs/howtos/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,6 @@
+=  Axon Framework Fundamentals
+:page-needs-improvement: intro, toc
+:page-needs-intro: Add an introduction section explaining the intention of the tutorial
+:page-needs-toc: Add table of contents
+
+
```

Comment:
Every content source needs a ROOT module. For now, I just added a home page. TBD if we want more info there.